extensions/wikia/Insights/models/InsightsQuerypageModel.php

```diff
@@ -82,6 +82,14 @@ public function arePageViewsRequired() {
 		return true;
 	}
 
+	public function isWlhLinkRequired() {
```

Comment:
Abbreviations makes it harder to use and search. Be explicit.
Also please provide a documentation.